docs/notebooks/fundamentals/computed-columns.ipynb

```diff
@@ -1879,9 +1879,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "pixeltable",
    "language": "python",
-   "name": "python3"
+   "name": "pixeltable"
```

Comment:
This seems to be breaking CI. You either need to run your notebooks under the `python3` kernelspec or we need to ensure `pixeltable` kernelspec is set up in CI.